### ISSUE TITLE
Customs smileys(from the latest msnp) aren't shown

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#311]
```
What steps will reproduce the problem?
1. Let somebody send you a custom smiley from the new WLM
2. ???
3. Profit

What is the expected output? What do you see instead?
The smiley.
The smiley key (like :3 instead of the actual smiley) 

What version of the product are you using? On what operating system?
Build from source(0.1.2) on Ubuntu 11.04

Please provide any additional information below.

This is most likely caused by the latest MSN protocol.
```

Original issue reported on code.google.com by `jerbo...@gmail.com` on 5 Jun 2011 at 5:45
- Blocked on: #320

**Comment by GoogleCodeExporter** [comment]
```
Oh and I would like to request all default WLM smileys(such as (L) and :S 
aren't included)
```

Original comment by `jerbo...@gmail.com` on 5 Jun 2011 at 6:02

**Comment by GoogleCodeExporter** [comment]
```
Donwload the original smiley pack from here...  (MSN, FACEBOOK AND MORE)
http://wiki.andreineculau.com/Original_Smileys_Theme_for_Pidgin
```

Original comment by `xion...@gmail.com` on 6 Jun 2011 at 1:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `devid...@gmail.com` on 25 Sep 2011 at 3:02

**Comment by GoogleCodeExporter** [comment]
```
Issue 322 has been merged into this issue.
```

Original comment by `devid...@gmail.com` on 14 Nov 2011 at 4:59
